ui/src/main/webapp/config/config.json

```diff
@@ -13,7 +13,31 @@
       }
     ]
   },
+  "time": {
```

Comment:
I don't think this 'time' section should be included in the default config. It is different for different schemas. Perhaps move it into the road-traffic config.json file. We also need to add a section to the README to explain how to configure the time config, e.g what time units are available.